share/zm-rpcapi.sysv

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+#
+### BEGIN INIT INFO
+# Provides:          zm-rpcapi
+# Required-Start:    $network $local_fs
+# Required-Stop:     $network $local_fs
+# Should-Start:      mysql postgresql
+# Should-Stop:       mysql postgresql
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: A JSON-RPC frontend for Zonemaster Backend
+# Description:       zm-rpcapi lets you add new tests and check for results in
+#                    the the Zonemaster Backend database
+### END INIT INFO
+
+BINDIR=${ZM_BACKEND_BINDIR:-/usr/local/bin}
+LOGFILE=${ZM_BACKEND_LOGDIR:-/var/log/zonemaster}/zm-starman-error.log
+PIDFILE=${ZM_BACKEND_PIDDIR:-/var/run/zonemaster}/zm-starman.pid
```

Comment:
I think it will be transparent to the users to let the default file names be "zonemaster_backend_rcpapi-error.log" and "zonemaster_backend_rcpapi.pid". And it would match the testagent file names better.

Reply:
I wanted to limit the scope of the change, but maybe this is a good time to change the file names after all.
Don't you think it would be better to make the files highlight their relation to the service script? I.e. naming the files like `zm-rpcapi.pid` etc.

Reply:
I agree. That is even better.